gh-badges/lib/index.js

```diff
@@ -1,15 +1,8 @@
 'use strict'
 
-const { makeBadge } = require('./make-badge')
-const { PDFKitTextMeasurer, QuickTextMeasurer } = require('./text-measurer')
+const makeBadge = require('./make-badge')
 
 class BadgeFactory {
-  constructor({ fontPath, fallbackFontPath, precomputeWidths = false }) {
```

Comment:
I know this isn't a backwards-incompatible change as such in that if we pass `fontPath`, `fallbackFontPath` or `precomputeWidths` nothing bad will happen - we will just ignore the values.

Do you reckon its worth also logging a deprecation message or something if we see one of those params passed here?

Reply:
Yea, adding a deprecation warning makes sense.